HotelBooking/HotelBooking.Tests/GuestServiceTests.cs

```diff
@@ -53,6 +56,7 @@ public async Task AddGuest_WhenEmailAlreadyExists_BadRequestException()
             Email = guestDto.Email,
             PhoneNumber = guestDto.PhoneNumber,
             DateBirth = guestDto.DateBirth,
+            PasswordHash = Convert.ToBase64String(hmac.ComputeHash(Encoding.UTF8.GetBytes("SomePassword")))
```

Comment:
użyłaś tej samej logiki w teście co i w implementacji, nasuwa się tutaj potrzeba wydzielenia tej logiki haskowania do dedykowanej klasy, może być też static klasa. Test nie musi znać dokładnego sposobu haskowania

Reply:
Jak Ania zmieni nazwę metody AddGuest to trzeba pamiętać też o zmianie nazw testów